protobuf/flow/executiondata/executiondata.proto

```diff
@@ -38,8 +38,46 @@ service ExecutionDataAPI {
   rpc SubscribeExecutionData(SubscribeExecutionDataRequest)
       returns (stream SubscribeExecutionDataResponse);
```

Comment:
please also add a deprecation notice similar to what was added for the old streaming events endpoint
https://github.com/onflow/flow/blob/e4cf6a528d3c848312eb7be959a9e8f6a580a5a7/protobuf/flow/executiondata/executiondata.proto#L66-L69